src/ui/components/EmptyStateTeaser.tsx

```diff
@@ -10,7 +10,7 @@ type Props = {
 
 export function EmptyStateTeaser({subheading, caption = 'Please check back for updates'}: Props) {
   return (
-    <View style={styles.container}>
+    <View style={styles.container} testID={'empty_state'}>
```

Comment:
Where is this used?

Reply:
This is not used in current PR, but we will need this to check for empty a state when in other PR.